### PR TITLE
Add metrics origin for kubeflow

### PR DESCRIPTION
**Pull request by HadhemiDD**
### What does this PR do?

Add metrics origin for kubeflow

### CODE REVIEW

**Review by vickenty — APPROVED** [review]

File: releasenotes/notes/Add-metrics-origins-for-Kubeflow-integration.-f024c8b4cd5e60a3.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Add metrics origins for Kubeflow integration.
```
Comment (buraizu) [inline releasenotes/notes/Add-metrics-origins-for-Kubeflow-integration.-f024c8b4cd5e60a3.yaml:11]:
```suggestion
    Add metric origins for the Kubeflow integration.
```

**Review by buraizu — APPROVED** [review]
Thanks, approving with a couple of minor updates requested

**Review by dkirov-dd — APPROVED** [review]

**Review by remeh — APPROVED** [review]
LGTM!

**Review by GustavoCaso — APPROVED** [review]

**Pull request merged**